UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -775,7 +777,7 @@ public void TestClassGeneric()
             {
                 var obj = objs[i];
                 Assert.Same(obj, TestObject);
-                Assert.Equal(TestObject.ThisPtr, objs[i].ThisPtr);
+                Assert.Equal(TestObject, objs[i]);
```

Comment:
how come the ThisPtr check was removed?

Reply:
There's no longer a `ThisPtr` property exposed because it makes the public surface area not match the WinMD contract and it isn't entirely accurate for composed types. There's no good reason to use the `ThisPtr` property, so I removed it early in the design.